.gitignore

```diff
@@ -11,6 +11,8 @@ moc_*.cpp
 ui_*.h
 qrc_*
 default/
+tests/mapreader/mapreader
+tests/staggeredrenderer/staggeredrenderer
```

Comment:
Yes that's true. But I'm just curious, as to how no one noticed this in a long time.